app/constants/__tests__/flagsEnv.spec.js

```diff
@@ -0,0 +1,28 @@
+import env from 'react-native-config';
+
+import { getBuildtimeFlags, parseFlags } from '../flagsEnv';
+
+describe('getBuildTimeFlags', () => {
+  it('returns the parsed flags', () => {
+    // Default value from __mocks__
```

Comment:
Consider setting a default value here, here to keep this test from breaking if others change the defaults.

Then the test is encapsulated to: given, when, then

The global mock is to unblock components that don't care but may use a `<FeatureFlag>` deeper in their tree.

Reply:
Good advice.

Reply:
Neat, didn't know it was this simple to unlock a single method from a file.

```js
jest.mock('../../constants/flagsEnv');
const { parseFlags } = jest.requireActual('../flagsEnv');
```